app/views/dams_objects/embed.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+   prefix = (defined?(componentIndex)) ? "component_#{componentIndex}_" : ''
```

Comment:
It looks like `componentIndex` hasn't defined. May need to retrieve it from `params['cmp']`.

Reply:
@lsitu I am not sure what you mean here, but I know it works. If `componentIndex` wasn't being defined, then the videos wouldn't load in the player and they load just fine. Maybe I am misunderstanding something? Please clarify. Thanks!

Reply:
I updated this code to use params['cmp'] as suggested. Fixed bug.